### ISSUE TITLE
Move to JuliaStats?

### ISSUE THREAD

**Issue by ParadaCarleton** [kshedden/Earth.jl#1]
Hi, first of all, I just wanted to thank you for this incredible package. It seems super useful!

Do you think it would make sense to move this package into the JuliaStats org, to make sure it stays up-to-date and easy to find? Thank you!

**Comment by kshedden** [comment]
Yes, that would be great.  If I can do anything to help let me know.

**Comment by ParadaCarleton** [comment]
Perfect! I think we'll probably need to make some revisions first, though; this code looks like great prototype code, but packages usually need to follow a slightly different style to make sure everything is fast, correct, and general.

I might make a PR for some of these, but I would recommend looking at [this](https://docs.google.com/document/d/1P3WduCaQ6u3KDDpY_7o94zE_gTLtJr0kHTx4PYdgkcs/edit?usp=sharing) as well as [this](https://modernjuliaworkflows.github.io/pages/writing/) for advice on package QA.
